ote/src/cljs/ote/views/transport_operator_ytj.cljs

```diff
@@ -24,7 +24,14 @@
             [ote.style.base :as style-base]
             [ote.ui.common :as uicommon]))
 
-(defn- delete-operator [e! operator]
+(defn- delete-operator [e! operator service-vector]
+  ;; When operator is passed from :transport-operators-to-save list they do not have business-id
+  (let [toggle-dialog (if (nil? (::t-operator/business-id operator))
```

Comment:
Instead of resolving toggle callback in delete-operator, could the caller pass the toggle callback as an argumen to this? That way there's no need to resolve it here.

Reply:
Yes. It is possible. And I didn't even consider this option when I was working with this. And passing a function might take less memory (not significant). But I think that right now I leave this as it is even it might be better solution... I'm not sure if it is. It might be and probably is but still :)